Sources/Logging/Locks.swift

```diff
@@ -26,12 +26,18 @@
 //
 //===----------------------------------------------------------------------===//
 
+#if canImport(WASILibc)
+// No locking on WASILibc
+#else
+
```

Comment:
This won't be needed if imports are structured the same way they are for other platforms.

```suggestion
```

Reply:
resolving all these because @MaxDesiatov approved